code_search/kubeflow/components/params.libsonnet

```diff
@@ -20,7 +20,7 @@
       eval_steps: 10,
       image: "gcr.io/kubeflow-examples/code-search:" + imageTag,
       imageGpu: "gcr.io/kubeflow-examples/code-search-gpu:" + imageTag,
-      dataflowImage: "gcr.io/kubeflow-examples/code-search-dataflow:v20181106-v0.2-76-g611636c-dirty-860631",
+      dataflowImage: "gcr.io/kubeflow-examples/code-search-dataflow:v20181109-dc79384",
```

Comment:
Did you build the image and push manually? Does kubeflow has release process to automatically update image tag?

(For /example project this might not matters that much, but just curious because /pipeline project faces this issue)

Reply:
In this particular case I built the image manually. Its possible to build images automatically by taking advantage of our CI/CD system.